Common/Data/Custom/TradingEconomics/TradingEconomicsEventFilter.cs

```diff
@@ -191,7 +191,7 @@ public static string FilterEvent(string eventName)
                 filteredName = SingleWordFilter(filteredName, kvp.Key, kvp.Value);
             }
 
-            return filteredName;
+            return filteredName.Trim();
```

Comment:
I think it seems we need at least two new tests for `FilterEvent`: for this line and the change bellow